cypress/e2e/read-your-lpa.cy.js

```diff
@@ -0,0 +1,25 @@
+describe('Check the LPA', () => {
```

Comment:
Should this be:

```suggestion
describe('Read your LPA', () => {
```
Or is this section a continuation of the check?